generic/system/khsp-cis-kubernetes-policies.yaml

```diff
@@ -0,0 +1,226 @@
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorHostPolicy
+metadata:
+  name: ksp-cis-deny-modification-of-api-server-config-files
+spec:
+  selector:
+    matchLabels:
+      node: master
+  action: Block
+  file:
+    matchPaths:
+    - path: /etc/kubernetes/manifests/kube-apiserver.yaml
+      readOnly: true
+    - path: /etc/kubernetes/pki/apiserver.*
```

Comment:
Do not specify pattern matching in the policies. Kubearmor does not have full support for it.

Reply:
@nyrahul Thank you for letting us know and the policy is still in revision and I'll revise the policy.